src/App.tsx

```diff
@@ -1,22 +1,24 @@
-import React from "react";
-import Home from "pages/Home";
```

Comment:
If possible, can you avoid these unwanted changes due to change in order of imports? It affects the readability of the pull request.

Reply:
Sorry for the delay. It has been fixed.